sdk/attestation/azure-security-attestation/tests/preparers.py

```diff
@@ -1,40 +1,83 @@
 # coding: utf-8
 
-#-------------------------------------------------------------------------
+# -------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
-#--------------------------------------------------------------------------
+# --------------------------------------------------------------------------
 
+from typing import List
 import pytest
 import functools
 from devtools_testutils import PowerShellPreparer
```

Comment:
Opportunities to save on imports
```suggestion
import functools
```